src/components/Card/README.md

```diff
@@ -17,6 +17,8 @@ keywords:
   - card with button in the footer
   - card with button in the heading
   - card with multiple sections
+  - card with multiple sections and subsections
```

Comment:
Curious, what are these `keywords` used for?

Reply:
They're used to index this page in our Algolia-powered search on the styleguide

Reply:
These are part of what the style guide parses and builds component pages with, in this case the examples are parsed into the example section, so adding these keywords helps folks find the right page if they're searching by example name alone in the style guide search bar.

<img width="583" alt="Screen Shot 2019-06-04 at 3 18 31 PM" src="https://user-images.githubusercontent.com/18447883/58907377-09643500-86dc-11e9-8130-4987678ea4db.png">